src/test/util/txmempool.cpp

```diff
@@ -36,3 +37,78 @@ CTxMemPoolEntry TestMemPoolEntryHelper::FromTx(const CTransactionRef& tx) const
 {
     return CTxMemPoolEntry{tx, nFee, TicksSinceEpoch<std::chrono::seconds>(time), nHeight, m_sequence, spendsCoinbase, sigOpCost, lp};
 }
+
+std::optional<std::string> CheckPackageMempoolAcceptResult(const Package& txns, const PackageMempoolAcceptResult& result, bool expect_valid,
+                                     const CTxMemPool* mempool)
```

Comment:
c1319395f74c0016b367e860050c0ff5a99dea85 nit: the whitespace here is weird, should probably align with parentheses

Reply:
realigned